R/zchunk_L123.bcoc_awb_R_S_T_Y.R

```diff
@@ -30,43 +30,78 @@ module_emissions_L123.bcoc_awb_R_S_T_Y_DISABLED <- function(command, ...) {
     # Load required inputs
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     L103.ag_Prod_Mt_R_C_Y_GLU <- get_data(all_data, "L103.ag_Prod_Mt_R_C_Y_GLU")
-    L121.AWBshare_R_C_Y_GLU <- get_data(all_data, "L121.AWBshare_R_C_Y_GLU")
+    L121.AWBshare_R_C_Y_GLU <- get_data(all_data, "temp-data-inject/L121.AWBshare_R_C_Y_GLU")
```

Comment:
Immediately transform this data into eventual form (once not `temp-data-inject`): `get_data() %>% gather(year, value, -..., -...) %>% mutate(year = as.integer(substr(year, 2, 5)))`. Sorry if this wasn't clear. I may make this change.